dev_guide/builds/build_inputs.adoc

```diff
@@ -853,22 +884,22 @@ into that directory, relative to your *_Dockerfile_* location. This makes the
 secret files available to the Docker build operation as part of the context
 directory used during the build.
 
-.Example of a Dockerfile referencing secret data
+.Example of a Dockerfile referencing secret and configMap data
 ====
 ----
 FROM centos/ruby-22-centos7
 
 USER root
 ADD ./secret-dir /secrets
-COPY ./secret2 /
+COPY ./config /
```

Comment:
not sure why one of these is doing ADD and the other COPY, let's make them consistent.